EIPS/eip-2678.md

```diff
@@ -0,0 +1,2132 @@
+`buildDependencies` section of the [Package
+Manifest](#term-package-manifest).
+
+<table>
+<colgroup>
+<col style="width: 50%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td><p>Required</p></td>
+<td><p>Yes</p></td>
+</tr>
+<tr class="even">
+<td><p>Type</p></td>
+<td><p>String</p></td>
+</tr>
+<tr class="odd">
+<td><p>Format</p></td>
+<td><p>See Below.</p></td>
+</tr>
+</tbody>
+</table>
+
+
+**Format**
+
+Values for this field **must** conform to *one of* the two formats
+herein.
+
+To reference a contract type from this Package, use the format
+`<contract-alias>`.
+
+-   The `<contract-alias>` value **must** be a valid [Contract
+    Alias](#term-contract-alias).
+
+-   The value **must** be present in the keys of the `contractTypes`
+    section of this Package.
+
+To reference a contract type from a dependency, use the format
+`<package-name>:<contract-alias>`.
+
+-   The `<package-name>` value **must** be present in the keys of the
+    `buildDependencies` of this Package.
+
+-   The `<contract-alias>` value **must** be be a valid [Contract
+    Alias](#term-contract-alias).
+
+-   The resolved package for `<package-name>` must contain the
+    `<contract-alias>` value in the keys of the `contractTypes` section.
+
+#### Address: `address`
+
+The `address` field defines the [Address](#term-address) of the
+[Contract Instance](#term-contract-instance).
+
+<table>
+<colgroup>
+<col style="width: 50%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td><p>Required</p></td>
+<td><p>Yes</p></td>
+</tr>
+<tr class="even">
+<td><p>Type</p></td>
+<td><p>String</p></td>
+</tr>
+<tr class="odd">
+<td><p>Format</p></td>
+<td><p>Hex encoded <code>0x</code> prefixed Ethereum address matching the regular expression <code>^0x[0-9a-fA-F]{40}$</code>.</p></td>
+</tr>
+</tbody>
+</table>
+
+
+#### Transaction: `transaction`
+
+The `transaction` field defines the transaction hash in which this
+[Contract Instance](#term-contract-instance) was created.
+
+<table>
+<colgroup>
+<col style="width: 50%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td><p>Required</p></td>
+<td><p>No</p></td>
+</tr>
+<tr class="even">
+<td><p>Type</p></td>
+<td><p>String</p></td>
+</tr>
+<tr class="odd">
+<td><p>Format</p></td>
+<td><p><code>0x</code> prefixed hex encoded transaction hash.</p></td>
+</tr>
+</tbody>
+</table>
+
+
+#### Block: `block`
+
+The `block` field defines the block hash in which this the transaction
+which created this *contract instance* was mined.
+
+<table>
+<colgroup>
+<col style="width: 50%" />
+<col style="width: 50%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td><p>Required</p></td>
+<td><p>No</p></td>
+</tr>
+<tr class="even">
+<td><p>Type</p></td>
+<td><p>String</p></td>
+</tr>
+<tr class="odd">
+<td><p>Format</p></td>
+<td><p><code>0x</code> prefixed hex encoded block hash.</p></td>
+</tr>
+</tbody>
+</table>
+
+
+#### Runtime Bytecode: `runtimeBytecode`
```

Comment:
What is the use case for this? Could this be replaced with `deploymentArguments` (ABI-encoded arguments used during deployment transaction)

Reply:
I believe it's also to support a deployment that references any combination of linked libraries (via `runtimeBytecode`).